nodejs/week3/homework/warmup/controllers/multiplicationController.js

```diff
@@ -0,0 +1,26 @@
+const multiply = (req, res) => {
+  const { firstParam, secondParam } =
+    "firstParam" in req.query ? req.query : req.body;
+
+  const first = Array.isArray(firstParam)
```

Comment:
You could have used the same pattern as in the subtraction controller here, what made you choose otherwise?